torchvision/prototype/transforms/functional/_color.py

```diff
@@ -7,10 +9,53 @@
 
 from torchvision.utils import _log_api_usage_once
 
-from ._meta import _num_value_bits, _rgb_to_gray, convert_dtype_image_tensor
+from ._meta import _num_value_bits, convert_dtype_image_tensor
 from ._utils import is_simple_tensor
 
 
+def _rgb_to_grayscale_image_tensor(
```

Comment:
Could you explain why we need this as public and private?

Reply:
The private `_rgb_to_grayscale_image_tensor()` is also used in other functionals (see below) with `preserve_dtype=False`, which isn't a parameter we'd like to expose. This is essentially just a renaming of the previous `_rgb_to_gray()` util (which also had a `cast` param that I renamed.)